docs/source/workflows/predictor_evaluation_workflows.rst

```diff
@@ -0,0 +1,264 @@
+Each workflow is composed of one or more :class:`PredictorEvaluators <citrine.informatics.predictor_evaluator.PredictorEvaluator>`.
+
+Predictor evaluators
+--------------------
+
+A predictor evaluator defines a method to evaluate a predictor and any relevant configuration, e.g. k-fold cross-validation evaluation that specifies 3 folds.
+Minimally, each predictor evaluator specifies a name, a set of predictor responses to evaluate and a set of metrics to compute for each response.
+Evaluator names must be unique within a single workflow (more on that `below <#execution-and-results>`__).
+Responses are specified as a set of strings, where each string corresponds to a descriptor key of a predictor output.
+Metrics are specified as a set of :class:`PredictorEvaluationMetrics <citrine.informatics.predictor_evaluation_metrics.PredictorEvaluationMetric>`.
+When defining an evaluator, the top-level metrics should be the union of all metrics computed across all responses.
+The evaluator will only compute the subset of metrics valid for each response.
+
+Cross-validation evaluator
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A :class:`~citrine.informatics.predictor_evaluator.CrossValidationEvaluator` performs k-fold cross-validation on a predictor.
+In addition to a name, set of responses to validate and metrics to compute, this evaluator defines the number of folds, number of trials and set of descriptor keys to ignore when grouping.
+The latter allows candidates with different values for the specified (ignored) keys and identical values all other predictor inputs to be placed in the same fold.
+
+Cross-validation can only be evaluated on predictors that define training data.
+During cross-validation, the predictor's training data is partitioned into k equally sized folds.
+Each fold acts as the test set once, and the remaining k-1 folds are used as training data.
+When the number of folds equals the number of training data points, the analysis is equivalent to leave-one-out cross-validation.
+Metrics are computed by comparing the model's predictions to observed values.
+
+Predictor evaluation metrics
+----------------------------
+
+Predictor evaluation metrics are defined as part of a :class:`~citrine.informatics.predictor_evaluator.PredictorEvaluator`.
+Available metrics depend on response type.
+For numeric responses, the following metrics are available:
+
+  - *Root-mean squared error* (:class:`~citrine.informatics.predictor_evaluation_metrics.RMSE`): square root of the average of the squared prediction error.
+    RMSE is a useful and popular statistical metric for model quality.
+    RMSE is optimized by least-squares regression, and in that sense is the most "natural" measure for it; it has the same units as the predicted quantity, and corresponds to the standard deviation of the variance not explained by the predictor.
```

Comment:
> the most "natural" measure for it
for what?

Reply:
I'm not sure what was originally meant by that. Let's take it out.